docs/contributing.md

```diff
@@ -130,7 +131,9 @@ Bazel is organized in several parts:
 6. The patch is exported as a Git commit, at which point the Gerrit code review
    is closed.
 
-We do not currently accept pull requests on GitHub.
+We do not currently accept pull requests for code contributions on GitHub.
```

Comment:
This is a recommendation, which is to permit non-code changes via pull-requests, to reduce impediments to contributing documentation fixes/changes. 

... like this one. :)